specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2021-11-01/healthcare-apis.json

```diff
@@ -0,0 +1,4087 @@
+              "$ref": "#/definitions/DicomServicePatchResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Patch request accepted and an existing resource is updated.",
+            "schema": {
+              "$ref": "#/definitions/DicomService"
+            }
+          },
+          "202": {
+            "description": "Accepted - Patch request accepted and an existing resource is being updated asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DicomService"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DicomServices"
+        ],
+        "description": "Deletes a DICOM Service.",
+        "operationId": "DicomServices_Delete",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete a dicomservice": {
+            "$ref": "./examples/dicomservices/DicomServices_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersion"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/dicomServiceName"
+          },
+          {
+            "$ref": "#/parameters/workspaceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource deleted."
+          },
+          "202": {
+            "description": "Delete request accepted; the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthcareApis/workspaces/{workspaceName}/iotconnectors": {
+      "get": {
+        "tags": [
+          "IotConnectors"
+        ],
+        "description": "Lists all IoT Connectors for the given workspace",
+        "operationId": "IotConnectors_ListByWorkspace",
+        "x-ms-examples": {
+          "List iotconnectors": {
+            "$ref": "./examples/iotconnectors/iotconnector_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
```

Comment:
could you keep the parameters as the order in the path ?

Reply:
This should apply to all the apis in the swagger

Reply:
@jianyexi  Thank you for the comment. Two follow up questions.
1. The apiVersion isn't a query parameter not a path parameter.  Does the comment still apply?
2. We have already published a Python SDK for Public Preview for these resources.  If I change the order of the API version I am assume it will reorder them in the SDKs generated causing breaking changes.  Is that not a concern?

Reply:
Okay, if the SDK already shipped , you can ignore my suggestion ,it only apply to new RP